.github/workflows/tests.yml

```diff
@@ -167,7 +173,7 @@ jobs:
         run: docker build -t brew --build-arg=version=16.04 .
 
       - name: Deploy the Docker image to GitHub Packages and Docker Hub
-        if: github.ref == 'refs/heads/master'
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
I think it'd still be good to have this run on pull requests too.

Reply:
This never runs on pull requests.

Reply:
Sorry, misread, yeh, it's building on pull requests which is what I wanted 👍🏻